### DISCUSSION TITLE
Display different widgets on same pages

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10606] [Q&A]
I create widget for header and footer need to display the header widget on header and footer widget on footer how i achieve this:
Once i call widget both of widget show in header as well on footer see below:

![issue](https://user-images.githubusercontent.com/5202291/139599648-3109030c-1620-4131-bae1-6b7fae38d2fe.png)

**Comment by ns8482e** [comment]
https://docs.orchardcore.net/en/latest/docs/reference/modules/Layers/#zones

**Reply by saleemhaq** [reply]
> https://docs.orchardcore.net/en/latest/docs/reference/modules/Layers/#zones
Not work I created new zone named named as newsletter and call it like @await RenderSectionAsync("newsletter", required: false) but it noting to display?

**Reply by saleemhaq** [reply]
Anyone help me out?
